.github/workflows/_python-tests.yml

```diff
@@ -55,52 +55,6 @@ jobs:
         shell: bash
         env:
           PROPERTY_TESTING_PRESET: ${{ inputs.property_testing_preset }}
-          CHROMA_RUST_BINDINGS_TEST_ONLY: "1"
-
-  test-rust-single-node-integration:
```

Comment:
why are you removing this?

Reply:
we can certainly add some smoke tests against the container, but running the full suite seems unnecessary
the Rust HTTP server fixture was added in https://github.com/chroma-core/chroma/pull/4037

Reply:
Yes I just said that ty though.

Reply:
cool I'll cut a ticket for container smoke tests